src/retrace/retrace.py

```diff
@@ -1355,6 +1323,38 @@ def convert_flattened_format(self):
         self._is_flattened_format = False
         return True
 
+    DUMP_LEVEL_PARSER = re.compile(r"^[ \t]*dump_level[ \t]*:[ \t]*([0-9]+).*$")
```

Comment:
It is customary to define member constants and variables at the top of the class (just after the `class Foo:` declaration). Can you please move it there?